tests/unit_test/callback/pyscript_handler_test.py

```
@@ -1435,13 +1422,18 @@ def test_add_schedule_job_http_failure(monkeypatch):
 
-    monkeypatch.setattr(CallbackUtility, "datetime_to_utc_timestamp", lambda dt: 1234567890)
+    monkeypatch.setattr(CallbackScriptUtility, "datetime_to_utc_timestamp", lambda dt: 1234567890)
 
     with pytest.raises(Exception) as exc_info:
-        CallbackUtility.add_schedule_job(schedule_action, date_time, data, tz, _id=_id, bot=bot, kwargs=kwargs_in)
+        CallbackScriptUtility.add_schedule_job(schedule_action, date_time, data, tz, _id=_id, bot=bot, kwargs=kwargs_in)
     assert "error message" in str(exc_info.value)
 
+import kairon.shared.pyscript.callback_pyscript_utils as mod
 
+def dummy_uuid7():
```

Ruff (0.8.2)

Redefinition of unused `dummy_uuid7` from line 1333

(F811)